### ISSUE TITLE
Get All Authors

### ISSUE THREAD

**Issue by MaggieChafee** [MaggieChafee/CapstoneServerSide#16]
# User Story

- When a user selects “Browse By Author”
- Then the page will load with all authors in the database, listed alphabetically by last name

# Acceptance Criteria
When this endpoint is called, it should return a list of all authors in the database. 

The HTTP response status code should be 200 when successfully run. 

Endpoint should be tested in Postman. 

# Dependencies
This is a stretch goal. All other MVP functionality needs to be completed first. 

# Dev Notes
This is a GET request. Here is an example GET request:
```csharp
app.MapGet("/orders", (HhpwDbContext db) =>
            {
                var allOrders = db.Orders.OrderBy(x => x.OrderOpen == false).ThenBy(x => x.Name).ToList();
                if (db.Orders == null)
                {
                    return Results.BadRequest();
                }
                return Results.Ok(allOrders);
            });
```
